webui/src/main/java/io/aegeus/ipfs/portal/AegeusContentHandler.java

```diff
@@ -380,7 +380,7 @@ private void actSend(HttpServerExchange exchange, VelocityContext context) throw
             new RedirectHandler("/portal/files?addr=" + rawFromAddr + "&error=" + e.getMessage() + "!").handleRequest(exchange);
         }
 
-        redirectHomePage(exchange);
+	new RedirectHandler("/portal/files?addr=" + rawFromAddr + "#pills-profile").handleRequest(exchange);
```

Comment:
you also have https://github.com/AegeusCoin/aegeus-ui/blob/master/webui/src/main/java/io/aegeus/ipfs/portal/AegeusContentHandler.java#L613